### PR TITLE
Test Tomn-uses/medium-less branch on staging

### PR DESCRIPTION
**Pull request by drew-usds**
Tests fixes for #137 https://github.com/orgs/usds/projects/77/views/4?layout=board&pane=issue&itemId=47032288 on staging branch.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-ruby '~> 2.7'
+ruby '~> 3.1.1'
```
Comment (TomNUSDS) [inline Gemfile:1]: Ruby v2->v3 is a little scary, but it it works then GREAT

I think the docker file (`deploy.yml`) needs to be ruby 3 as well.
`ruby-version: '2.7' # Not needed with a .ruby-version file`

Also the documentation?
documentation/2023-refresh/engineering-resources.md

Basically, I just searched for `ruby` and noticed these places.

File: _layouts/default.html [hunk 2]
```diff
@@ -93,7 +95,7 @@
       <section class="site-c-billboard">
         <div class="grid-container">
           <p>We need you.</p>
-          <p>Let’s help millions of people together.</p>
+          <p>Let's help millions of people together.</p>
```
Comment (TomNUSDS) [inline _layouts/default.html]: This smart quote `’` to regular quote `'` might be a mistake.

**Review by TomNUSDS — APPROVED** [review]
lgtm. Only thing was updating to Ruby v3 has other references we should update?

Still working on fixing the `visual-refresh` script.

File: Gemfile (hunk 1 above)
Comment (drew-usds) [reply Gemfile:1]: Updated references: https://github.com/usds/website/pull/917/commits/f2216ca075b611562ffbbe6e9f006167fbbdec22

**Pull request merged**